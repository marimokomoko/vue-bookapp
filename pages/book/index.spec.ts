import { createLocalVue, mount, Wrapper } from '@vue/test-utils'
import Vuetify from 'vuetify'
import Index from '@/pages/book/index.vue'
let wrapper: Wrapper<Vue>;

describe('画面レイアウト', () => {
    it('スナップショットテスト', () => {
        wrapper = mount(Index, {
            localVue: createLocalVue(),
            vuetify: new Vuetify(),
            stubs: ['router-link']
        })
        expect(wrapper.html()).toMatchSnapshot()
    })
})