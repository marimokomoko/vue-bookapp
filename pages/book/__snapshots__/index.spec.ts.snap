// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`画面レイアウト スナップショットテスト 1`] = `
"<div>
  <h1>BookIndex</h1>
  <div class=\\"row\\">
    <div class=\\"col col-8\\">
      <router-link-stub class=\\"mb-3 v-btn v-btn--is-elevated v-btn--has-bg v-btn--router theme--light v-size--default primary\\"><span class=\\"v-btn__content\\">検索する</span></router-link-stub>
    </div>
    <div class=\\"col col-4\\"><button type=\\"button\\" class=\\"v-btn v-btn--is-elevated v-btn--has-bg theme--light v-size--default error\\"><span class=\\"v-btn__content\\"> 削除する </span></button></div>
  </div>
  <div class=\\"row\\"></div>
</div>"
`;
